front/Dockerfile

```
@@ -26,16 +23,16 @@ FROM nginx AS staging-deploy
 COPY nginx.conf /etc/nginx/conf.d/default.conf
 COPY --from=staging-builder /app/dist /usr/share/nginx/html
 
-FROM node:22-bullseye-slim AS production-builder
+FROM oven/bun:${BUN_VERSION}-slim AS production-builder
 WORKDIR /app
 
-RUN npm install -g bun@v1.1.22
-
 COPY package.json bun.lockb ./
 RUN bun install --frozen-lockfile
```

これでイメージサイズを小さくできないかな（productionも同じ）
```suggestion
RUN bun install --frozen-lockfile --production
```